context.go

```diff
@@ -313,9 +313,19 @@ func checkRequiredFlags(flags []Flag, context *Context) requiredFlagsErr {
 	var missingFlags []string
 	for _, f := range flags {
 		if rf, ok := f.(RequiredFlag); ok && rf.IsRequired() {
-			key := strings.Split(f.GetName(), ",")[0]
-			if !context.IsSet(key) {
-				missingFlags = append(missingFlags, key)
+			key := strings.Split(f.GetName(), ",")
+			shortName := strings.TrimSpace(key[0])
+			if len(key) > 1 {
```

Comment:
Wouldn't it be better to just iterate over the split result here?

Reply:
If I iterate over the values, I would get false for names and the loop would most likely exit. I need to check both of them together to ensure that the conditions are met and we are very sure of the flag being absent